components/eam/src/physics/cam/aer_rad_props.F90

```diff
@@ -77,7 +79,21 @@ subroutine aer_rad_props_init()
         'EXTINCT LW H2O window band 7 output directly read from prescribed input file', flag_xyfill=.true.)
    call addfld ('extinct_sw_inp',(/ 'lev' /),    'A','1/km',&
         'Aerosol extinction directly read from prescribed input file', flag_xyfill=.true.)
-
+   ! add full bands radiation output
+   ! longwave: ext_sao_lw (pcols,pver,nlwbands)
+   ! ext_sao_sw, ssa_sao, af_sao (pcols,pver,nswbands)
+   !if (is_output_interactive_volc) then
+   !call add_hist_coord('nlwbands',    nlwbands,    'NLWBANDS')  
+   !call add_hist_coord('nswbands',    nswbands,    'NSWBANDS')   
+   !call addfld ('ext_sao_lw',(/ 'lev     ', 'nlwbands' /),    'A','1/m',&
+   !     'Aerosol LW radiation properties comparable to prescribed input ext_earth', flag_xyfill=.true.) 
+   !call addfld ('ext_sao_sw',(/ 'lev     ', 'nswbands' /),    'A','1/m',&
+   !     'Aerosol SW radiation properties comparable to prescribed input ext_sun', flag_xyfill=.true.) 
+   !call addfld ('ssa_sao',(/ 'lev     ', 'nswbands' /),    'A','1/m',&
+   !     'Aerosol SW radiation properties comparable to prescribed input omega_sun', flag_xyfill=.true.) 
+   !call addfld ('af_sao',(/ 'lev     ', 'nswbands' /),    'A','1/m',&
+   !     'Aerosol SW radiation properties comparable to prescribed input g_sun', flag_xyfill=.true.) 
+   !endif
```

Comment:
If they are just some temporary output. Since those are commented out, maybe we can remove them. If it is some diagnostics we want to add, we can either uncomment them or make a separate PR.

Reply:
this is not necessary output for now. You are right, I can add it later